src/components/organize/tasks/TaskList/index.tsx

```diff
@@ -102,6 +76,17 @@ const TaskList: React.FunctionComponent<TaskListProps> = ({ tasks }) => {
       </List>
     </Card>
   );
+
+  function renderTaskList(status: TASK_STATUS) {
+    return (
+      status in tasksGroupedByStatus && (
+        <TaskStatusSublist
+          status={status}
+          tasks={tasksGroupedByStatus[status]}
+        />
+      )
+    );
+  }
 };
```

Comment:
I don't really think we need to refactor this, just rename the sublist bit for `CLOSED` to `PASSED` and move it out. the diff would be way smaller

Reply:
I unfortunately don't understand what this means - it's probably quicker (if you don't mind) for you to jump in and do your suggested refactor, rather than try to explain it to me in a different way ❤️